packages/ui/src/helpers/directives.ts

```diff
@@ -0,0 +1,15 @@
+import { ObjectDirective } from "vue";
+
+export const clickOutside : ObjectDirective = {
```

Comment:
We should probs look at using VueUse it has loads of utils like this

Reply:
I was gonna bring in VueUse but didn't know if it would increase the bundle massively or not